docs/flashbots-mev-boost/license.md

```diff
@@ -0,0 +1,7 @@
+The code in this project is free software under the [MIT License](https://github.com/flashbots/mev-boost/blob/main/LICENSE).
```

Comment:
do we need/want this `license.md` chapter?

Reply:
removed, not needed on docs.flashbot